README.md

```diff
@@ -27,7 +27,7 @@ dependencies:
   artemis: '>=2.0.0 <3.0.0' # only if you're using ArtemisClient!
   json_serializable: ^3.0.0
   equatable: ^0.6.1
-  gql: '>=0.7.3 <1.0.0'
```

Comment:
`gql` is still necessary because of the query code artemis generates (actually the `ast.dart` file)

Reply:
Strange things happens. Instead of duplicating and renaming that line I just renamed it)))